src/components/RichTextEditor/RichTextEditor.spec.tsx

```diff
@@ -31,7 +31,20 @@ const RichTextWithClearButton: FC<Pick<RichTextEditorProps, "value">> = ({ value
     );
 };
 
+function ignoreResizeObserverLoopErr(): Mocha.Func {
+    return () => {
+        const regExResizeObserverLoopErr = /^[^(ResizeObserver loop limit exceeded)]/;
+        cy.on("uncaught:exception", (err) => {
+            if (regExResizeObserverLoopErr.test(err.message)) {
+                return false;
+            }
+        });
+    };
+}
```

Comment:
Sam fixed it with a `setTimeout`: https://github.com/Frontify/arcade/pull/724

Reply:
applied